src/test/java/task70/util/SingleWebDriver.java

```diff
@@ -42,6 +47,54 @@ public WebDriver getRemoveWebDriver() throws MalformedURLException {
         return driver;
     }
 
+    public RemoteWebDriver getSauceLabsWebDriver() throws MalformedURLException {
+        MutableCapabilities multcaps = null; //= new MutableCapabilities();
+        switch (TEST_CONFIGURATION)
+        {
+            case "Windows 10 Microsoft Edge latest":
```

Comment:
It is better to make name of the case shorter and in uppercase letters. WIN10 could be enough